ThunderCloud/QuizAreaSelectionViewController.swift

```diff
@@ -31,15 +31,23 @@ class QuizAreaSelectionViewController: UIViewController {
 		guard let question = question else {
 			return
 		}
-		
-		imageView.image = question.selectionImage
-		let imageAspect = question.selectionImage.size.height / question.selectionImage.size.width
-		heightConstraint.constant = imageAspect * imageView.frame.width
-		
+        
 		let imageAnalyser = ImageColorAnalyzer(image: question.selectionImage)
 		imageAnalyser.analyze()
 		circleColor = imageAnalyser.detailColor ?? .black
 	}
+    
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Does this work in `viewWillAppear`? The user may see the frame changing in `viewDidAppear` :D

Reply:
This change was made in https://github.com/3sidedcube/ThunderCloud/pull/148, as the aspect ratio was being calculated incorrectly while the code was within `viewWillAppear` :)